### ISSUE TITLE
CORS breaks visualize.cube

### ISSUE THREAD

**Issue by jbe456** [atoti/atoti#31]
## Description

In version `0.4.0.20200507160709`, I'm unable to run `cube.visualize`. Furthermore, any api calls after that seems to hang forever.

It seems to be a CORS issue:
```
Access to fetch at 'http://localhost:34009/versions/rest' from origin 'http://localhost:8888' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
```

## Reproduce

Take any notebook and try to run `cube.visualize`. I reproduced against a clean environment on several notebooks.

![image](https://user-images.githubusercontent.com/4415696/81327907-4c711a00-909c-11ea-8ed3-a432cb9c32fb.png)

## Context

- Atoti: 0.4.0.20200507160709
- Python: 3.8.2
- Operating System: linux

**Comment by fabiencelier** [comment]
Hello,
I cannot reproduce this issue in the latest version.
I have tried with chrome 81 and firefox 76, what's your browser ?
Do you have any additional info or warnings in your browser console ?
Can you check if the application works outside jupyter lab ?

**Comment by fabiencelier** [comment]
I could not reproduce because I had `jupyter_server_proxy` installed that allow ActiveUI to be on the same port as jupyterlab so no issue with CORS

With the proxy disabled (`jupyter serverextension disable --sys-prefix jupyter_server_proxy`) I do reproduce your issue.

We will investigate, thanks for reporting !

**Comment by fabiencelier** [comment]
The CORS issue has been fixed in 0.4.0
